### PR TITLE
add site config for SGMS

### PR DESCRIPTION
**Pull request by gatoravi**
this addresses the issue in #527

### CODE REVIEW

File: lib/perl/Genome/Site/SGMS.t [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env genome-perl
+
+use strict;
+use warnings;
+
+use above 'Genome';
+use Test::More tests => 1;
+
+# This test was auto-generated because './Site/TGI.pm'
```
Comment (tmooney) [inline lib/perl/Genome/Site/SGMS.t]: A minor nitpick: this isn't `./Site/TGI.pm`.

**Comment by tmooney** [comment]
+1 :necktie:

File: lib/perl/Genome/Site/SGMS.t (hunk 1 above)
Comment (gatoravi) [inline lib/perl/Genome/Site/SGMS.t]: fair enough, fixed this :pizza:

**Comment by acoffman** [comment]
+1

**Pull request merged**